doc/manuals/orion-api.md

```diff
@@ -1798,79 +1808,65 @@ Content-Type is `application/json`
+| `expires`      | ✓        | string | Subscription expiration date in ISO8601 format. Permanent subscriptions must omit this field. |
+| `status`       |          | string | Either `active` (for active subscriptions) or `inactive` (for inactive subscriptions). If this field is not provided at subscription creation time, new subscriptions are created with the `active` status, which can be changed by clients afterwards. For expired subscriptions, this attribute is set to `expired` (no matter if the client updates it to `active`/`inactive`). Also, for subscriptions experiencing problems with notifications, the status is set to `failed`. As soon as the notifications start working again, the status is changed back to `active`.                                                                                              |
+| `throttling`   | ✓        | number | Minimal period of time in seconds which must elapse between two consecutive notifications.    |
+
+#### `subscription.subject`
 
 A `subject` contains the following subfields:
 
-+ `entities`: A list of objects, each one composed of the following subfields:
-  + `id` or `idPattern`: Id or pattern of the affected entities. Both cannot be used at the same
-     time, but one of them must be present.
-  + `type` or `typePattern`: Type or type pattern of the affected entities. Both cannot be used at
-     the same time. If omitted, it means "any entity type".
-+ `condition`: Condition to trigger notifications. This field is optional and it may contain two
-  properties, both optional:
-    + `attrs`: array of attribute names
-    + `expression`: an expression composed of `q`, `mq`, `georel`, `geometry` and `coords` (see "List
-       entities" operation above about this field).
+| Parameter      | Optional | Type   | Description                                                                                   |
+|----------------|----------|--------|-----------------------------------------------------------------------------------------------|
+| `entities`     | ✓        | string | A list of objects, each one composed of the following subfields: <ul><li><code>id</code> or <code>idPattern</code> Id or pattern of the affected entities. Both cannot be used at the same time, but one of them must be present.</li> <li><code>type</code> or <code>typePattern</code> Type or type pattern of the affected entities. Both cannot be used at the same time. If omitted, it means "any entity type".</li></ul> |
+| `condition`    | ✓        | string | Condition to trigger notifications. This field is optional and it may contain two properties, both optional: <ul><li><code>attrs</code> Array of attribute names that will trigger the notification. </li> <li><code>expression</code> An expression composed of <code>q</code>, <code>mq</code>, <code>georel</code>, <code>geometry</code> and <code>coords</code> (see "List entities" operation above about this field)</li></ul> |
```

Comment:
```suggestion
| `entities`     | ✓        | array| A list of objects, each one composed of the following subfields: <ul><li><code>id</code> or <code>idPattern</code> Id or pattern of the affected entities. Both cannot be used at the same time, but one of them must be present.</li> <li><code>type</code> or <code>typePattern</code> Type or type pattern of the affected entities. Both cannot be used at the same time. If omitted, it means "any entity type".</li></ul> |
| `condition`    | ✓        | object| Condition to trigger notifications. This field is optional and it may contain two properties, both optional: <ul><li><code>attrs</code> Array of attribute names that will trigger the notification. </li> <li><code>expression</code> An expression composed of <code>q</code>, <code>mq</code>, <code>georel</code>, <code>geometry</code> and <code>coords</code> (see "List entities" operation above about this field)</li></ul> |
```